### PR TITLE
Some ui fixes during hornet engineering time

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose

- No ticket is linked to this PR.

### Changes

- Fix Dashboard.
- Fix several places that still have "Zero".
- Remove redundant code and Google Auth button.
- Update the API `/version` endpoint to separate pipeline-versions and system versions.

### Review Instructions

- No instructions.

### CODE REVIEW

File: ui/src/router/index.js [hunk 1]
```diff
@@ -16,23 +16,23 @@ const routes = [
     // route level code-splitting
     // this generates a separate chunk (about.[hash].js) for this route
```
Comment (samanehsan) [inline ui/src/router/index.js:17]: Oh there's another reference to "about". Can you update it to say (query.[hash].js)?

Comment (rexwangcc) [reply ui/src/router/index.js:17]: Oh, good catch!!! Thanks, I will address it right now.

File: src/zero/api/routes.clj [hunk 2]
```diff
@@ -121,10 +122,14 @@
...
-                                    :build    pos-int?
-                                    :built    string?}}}
+            :handler (handlers/success (let [versions (zero/get-the-version)
+                                             pipeline-versions-keys (keep (fn [x] (when (and (string? x) 
+                                                                                             (str/ends-with? x ".wdl")) x)) 
+                                                                          (keys versions))]
+                                         {:pipeline-versions (select-keys versions pipeline-versions-keys)
+                                          :version (apply dissoc versions pipeline-versions-keys)}))
+            :responses {200 {:body {:version map?
+                                    :pipeline-versions map?}}}
```
Comment (tbl3rd) [inline src/zero/api/routes.clj:132]: Seems like the wrong place to separate these.
Either do it when constructing the version itself,
or do it in the UI, which is the code that cares.
Also, map? is too loose.
We want /version to at least return a :version.

Comment (rexwangcc) [reply src/zero/api/routes.clj:132]: > map? is too loose.

That makes sense, will fix it in a follow up PR.

> Either do it when constructing the version itself,
or do it in the UI, which is the code that cares.

The problem I had with doing this in the frontend is that we have to use specific getters like: https://github.com/broadinstitute/wfl/pull/28/files#diff-88ce83892fa39769e0067bd6efa72316L35 which will require updates every time we add or update a pipeline (name). If that's fine with you I could revert it!

**Pull request merged**